src/Core/Connector/HttpConnector.js

```diff
@@ -37,8 +38,12 @@ function HttpConnector(
 ) {
   /* eslint-disable no-param-reassign */
   axios.defaults.paramsSerializer = paramsSerializer;
-  axios.defaults.transformRequest.push(requestSerializer);
-  axios.defaults.transformResponse.push(responseNormalizer);
+  const transformRequest = cloneDeep(axios.defaults.transformRequest);
```

Comment:
Ik denk eerder dat we een nieuwe instance van Axios moeten maken?

const axiosInstance = axios.create({...axios.defaults, transformResponse: de rest}

Dan hebben we een eigen instance op basis van wat doorgegeven is (door gebruik van axios.defaults neem je de instellingen van de doorgegeven instance mee)

Reply:
Lijkt me een goed idee. Ik ga kijken of dat inderdaad goed werkt.

Reply:
Ik heb verschillende manieren uitgeprobeerd om het zo aan de praat te krijgen, maar dat werkt toch niet. Althans, ik krijg het niet aan de praat.
Ik denk dat dit toch de manier is om het te doen.